airbyte-integrations/connectors/destination-motherduck/destination_motherduck/spec.json

```
@@ -0,0 +1,34 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/motherduck",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Destination Duckdb",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": true,
+    "properties": {
+      "motherduck_api_key": {
+        "title": "MotherDuck API Key",
+        "type": "string",
+        "description": "API key to use for authentication to a MotherDuck database.",
+        "airbyte_secret": true
+      },
+      "destination_path": {
+        "title": "Destination DB",
+        "type": "string",
+        "description": "Path to the .duckdb file, or the text 'md:' to connect to MotherDuck. The file will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/duckdb\">docs</a>",
+        "examples": ["/local/destination.duckdb", "md:", "motherduck:"]
+      },
+      "schema": {
+        "title": "Destination Schema",
+        "type": "string",
+        "description": "Database schema name, default for duckdb is 'main'.",
+        "example": "main"
+      }
+    }
+  },
+  "supportsIncremental": true,
+  "supportsNormalization": true,
+  "supportsDBT": true,
+  "supported_destination_sync_modes": ["overwrite", "append"]
```

Note: We should try to add the dedupe mode here as well. If not in this iteration, then in a subsequent release.

---

Yes we should support this. What happens when I add it? Does CI kick something off?

---

Replied in Slack. This flag tells the platform that we know how to handle specific write mode values passed along with the catalog.

---

Update (also posted to Slack): The code you added in the CDK seems to correctly handle this modes, and it looks like you are correctly getting the DestinationSyncMode per stream from the configured catalog.

We should be good to add the dedupe sync type. 

```suggestion
  "supported_destination_sync_modes": ["overwrite", "append", "append_dedup"]
```